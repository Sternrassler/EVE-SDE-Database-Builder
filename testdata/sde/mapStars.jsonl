{"starID":40000000,"solarSystemID":30000001,"typeID":3800,"x":0.0,"y":0.0,"z":0.0,"radius":1.4e9}
{"starID":40000141,"solarSystemID":30000142,"typeID":3802,"x":0.0,"y":0.0,"z":0.0,"radius":1.6e9}
{"starID":40000200,"solarSystemID":30000003,"typeID":3801,"x":0.0,"y":0.0,"z":0.0,"radius":1.2e9}
