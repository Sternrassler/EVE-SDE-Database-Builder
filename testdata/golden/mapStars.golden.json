[
  {
    "starID": 40000000,
    "solarSystemID": 30000001,
    "typeID": 3800,
    "radius": 1400000000,
    "temperature": null,
    "luminosity": null
  },
  {
    "starID": 40000141,
    "solarSystemID": 30000142,
    "typeID": 3802,
    "radius": 1600000000,
    "temperature": null,
    "luminosity": null
  },
  {
    "starID": 40000200,
    "solarSystemID": 30000003,
    "typeID": 3801,
    "radius": 1200000000,
    "temperature": null,
    "luminosity": null
  }
]